notary-api/src/jobs/paymentsHelpers.js

```diff
@@ -0,0 +1,23 @@
+import R from 'ramda';
+
+const reformatNotarizations = (result) => {
+  const reformat = R.pipe(
+    R.groupBy(R.prop('id')), // Object<BatPayId, Array<Pair<BatPayId,Address>>>
+    R.map(R.pluck('address')), // Object<BatPayId, Array<Address>>
+    R.mapObjIndexed(addresses => ({ [result]: addresses })),
+    // Object<BatPayId, Object<result,Array<Address>>>
+  );
+
+  return sellers => (sellers ? reformat(sellers) : {});
+};
+
+const mergeByBatPayId = (id, left, right) => ({ ...left, ...right });
```

Comment:
We can omit the `mapObjIndexed ` if we just.
```suggestion
const mergeByBatPayId = (id, completed, rejected) => ({ completed, rejected });
```
And then we won't need `reformatNotarizations` we can just use `reformat` instead.